### PR TITLE
Sprint 2

### PR DESCRIPTION
**Pull request by surajsbharadwaj**
### Description

Please include a summary of the change and which issue is fixed. Also, please include the motivation and context for the change. List any dependencies that are required for this change.

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [ ] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

File: examples/ibm-catalog/standard-solution/main.tf [hunk 1]
```diff
@@ -26,12 +26,18 @@ provider "ibm" {
   ibmcloud_api_key = var.ibmcloud_api_key != null ? var.ibmcloud_api_key : null
 }
 
+locals {
+  location = regex("us-south|us-east|eu-de|eu-gb", var.prerequisite_workspace_id)
+}
```
Comment (ocofaigh) [inline examples/ibm-catalog/standard-solution/main.tf:31]: what happens when schematics add support for more regions? We need to maintain the list here. Can we future proof this and return the string before the first dot?
@argeiger had proposed a regex that could work for this

File: examples/ibm-catalog/standard-solution/variables.tf [hunk 2]
```diff
@@ -1,4 +1,4 @@
-variable "slz_workspace_id" {
+variable "prerequisite_workspace_id" {
```
Comment (ocofaigh) [inline examples/ibm-catalog/standard-solution/variables.tf:1]: renaming variables means we need to bump the major version. OK to do this?

**Comment by ocofaigh** [comment]
@surajsbharadwaj I left some comments. Also there is a pre-commit hook is failing in the pipeline. Please run locally and fix (you may need to pull in latest common-dev-assets and run `make` to pull latest dependency versions). 
Also code needs to be rebased with main branch

File: examples/ibm-catalog/standard-solution/main.tf (hunk 1 above)
Comment (surajsbharadwaj) [reply examples/ibm-catalog/standard-solution/main.tf:31]: let me check. Just back from vacation. Is it on slack channel ?

**Pull request closed without merging**